lib/helpers/itemHelpers.js

```diff
@@ -15,6 +15,12 @@ if (Meteor.isClient) {
 				};
 			}
 		},
+    isSaved: function(){
+      var myId = Meteor.userId();
+      var query = Saves.find(myId, {fields: {item_id:true} }).fetch()[0].item_id;
```

Comment:
Line is too long.<br>Identifier 'item_id' is not in camel case.<br>'Saves' is not defined.